tasks/coverage/src/runtime/getBabelHelpers.js

```diff
@@ -1,122 +1,124 @@
-// Bundle this with `rolldown` to get `babelHelpers.js`
+// Run `npx rolldown ./getBabelHelpers.js > babelHelpers.js`
+// to bundle this with `rolldown` and get`babelHelpers.js`
+
 function getBabelHelpers() {
   const babel = {};
-  babel.applyDecoratedDescriptor = require('@babel/runtime/helpers/applyDecoratedDescriptor');
-  babel.applyDecs = require('@babel/runtime/helpers/applyDecs');
-  babel.applyDecs2203 = require('@babel/runtime/helpers/applyDecs2203');
-  babel.applyDecs2203R = require('@babel/runtime/helpers/applyDecs2203R');
-  babel.applyDecs2301 = require('@babel/runtime/helpers/applyDecs2301');
-  babel.applyDecs2305 = require('@babel/runtime/helpers/applyDecs2305');
-  babel.applyDecs2311 = require('@babel/runtime/helpers/applyDecs2311');
-  babel.arrayLikeToArray = require('@babel/runtime/helpers/arrayLikeToArray');
-  babel.arrayWithHoles = require('@babel/runtime/helpers/arrayWithHoles');
-  babel.arrayWithoutHoles = require('@babel/runtime/helpers/arrayWithoutHoles');
-  babel.assertClassBrand = require('@babel/runtime/helpers/assertClassBrand');
-  babel.assertThisInitialized = require('@babel/runtime/helpers/assertThisInitialized');
-  babel.asyncGeneratorDelegate = require('@babel/runtime/helpers/asyncGeneratorDelegate');
-  babel.asyncIterator = require('@babel/runtime/helpers/asyncIterator');
-  babel.asyncToGenerator = require('@babel/runtime/helpers/asyncToGenerator');
-  babel.awaitAsyncGenerator = require('@babel/runtime/helpers/awaitAsyncGenerator');
-  babel.AwaitValue = require('@babel/runtime/helpers/AwaitValue');
-  babel.callSuper = require('@babel/runtime/helpers/callSuper');
-  babel.checkInRHS = require('@babel/runtime/helpers/checkInRHS');
-  babel.checkPrivateRedeclaration = require('@babel/runtime/helpers/checkPrivateRedeclaration');
-  babel.classApplyDescriptorDestructureSet = require('@babel/runtime/helpers/classApplyDescriptorDestructureSet');
-  babel.classApplyDescriptorGet = require('@babel/runtime/helpers/classApplyDescriptorGet');
-  babel.classApplyDescriptorSet = require('@babel/runtime/helpers/classApplyDescriptorSet');
-  babel.classCallCheck = require('@babel/runtime/helpers/classCallCheck');
-  babel.classCheckPrivateStaticAccess = require('@babel/runtime/helpers/classCheckPrivateStaticAccess');
+  babel.applyDecoratedDescriptor = require('../../babel/packages/babel-helpers/lib/helpers/applyDecoratedDescriptor');
+  babel.applyDecs = require('../../babel/packages/babel-helpers/lib/helpers/applyDecs');
```

Comment:
Use babel-helpers rather than babel-runtime as the `@babel/runtime` helpers were generated by `babel-helpers/lib/helpers`